docs/team/lithops-zty.md

```diff
@@ -0,0 +1,46 @@
+---
+  layout: default.md
+  title: "John Doe's Project Portfolio Page"
```

Comment:
might want to put your name here instead of `John Doe`

Reply:
thanks, will do